src/main/java/seedu/address/ui/CompletableDeadlineCard.java

```diff
@@ -27,6 +27,18 @@
     @FXML
     private Label completedLabel;
 
+    /**
+     * Creates a {@code CompletableDeadlineCard} with the given {@code CompletableDeadline} without an index to display.
+     */
+    public CompletableDeadlineCard(CompletableDeadline deadline) {
+        super(FXML);
+        this.deadline = deadline;
+        id.setText("");
```

Comment:
Looks like id isn't being used at all.

Reply:
In reference to the previous PR haha

Reply:
ID is not used in the overview section.

Reply:
So each card has 2 constructors, one with an ID (if we need to display the ID so user can select it) and one without, for  the overview section where no id is required. 

Why not have ID's for both?
I'm afraid users will get confused. I also don't want the users to be able to run commands on tasks in the overview section, as this might lead to confusion.

Reply:
I see. That does make sense and simplifies a lot of things!